package.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>gz-physics7</name>
+  <version>7.2.0</version>
+  <description>Gazebo Physics : Physics classes and functions for robot applications</description>
+  <maintainer email="scpeters@openrobotics.org">Steve Peters</maintainer>
+  <license>Apache License 2.0</license>
+  <url type="website">https://github.com/gazebosim/gz-physics</url>
+
+  <buildtool_depend>cmake</buildtool_depend>
+
+  <build_depend>gz-cmake</build_depend>
```

Comment:
```suggestion
  <build_depend>gz-cmake3</build_depend>
```

Reply:
Fixed in https://github.com/gazebosim/gz-physics/pull/608/commits/df390df0f936bc5cacb5bfd15d52e49226763eb8